### DISCUSSION TITLE
Problem noticed in ssr (while using third-party library)

### DISCUSSION THREAD

**Discussion by limjunhyuk97** [sveltejs/kit#6961] [Q&A] [answered]
Hi! I'm currently using svele-kit and I found some issue in SSR process.

<img width="1434" alt="image" src="https://user-images.githubusercontent.com/59442344/191699189-b07efbbb-ba7a-4493-af1a-802ef97166bc.png">

To give some details about the situation happened above, 
- I was using 'bytemd' to embed md file editor in my project
- There was no problem while using that library making project in my local (no problem detected in 'npm run dev')
- But after deploying it to netlify, every time i reload the page which imports 'bytemd' component, the problem occurs
- And i found out that bytemd has dependency in 'codemirror' library

So what i tried is 
- diagnosing that the 'vite' bundler was a problem, I tried using various plugins to convert commonJS 'require' statement into ES 'import' statement in dependency pre-bundling process but it didn't work.

Figure below is the error message that I found in sources tab
<img width="740" alt="image" src="https://user-images.githubusercontent.com/59442344/191701002-63b65031-d006-4040-87e0-aa1af72e22a3.png">

So now I'm thinking about some reasons that might be caused by svelte kit..? (Or not.. LOL..)
It would be so nice of you to give me some pointers about this..!

Thank you so much for reading my question, and I hope svelte and svelte-kit to become more powerful day after day.

**Comment by dummdidumm** [comment, marked as answer]
Are you using the latest version of the Netlify adapter? That adapter used to have problems with ESM imports, but it should be fixed now ( https://github.com/sveltejs/kit/issues/5727 )

**Reply by limjunhyuk97** [reply]
THANK YOU SO MUCH.... YOU SAVED MY DAY... 
I was stucked in this problem and finding solutions for like a day.. 
GOD BLESS YOU!!
